iac-extensions/arm/src/main/java/org/sonar/iac/arm/checks/utils/CheckUtils.java

```diff
@@ -73,4 +76,8 @@ public static List<Tree> resolveProperties(Queue<String> path, Tree tree) {
     }
     return tree != null ? List.of(tree) : Collections.emptyList();
   }
+
+  public static Predicate<Expression> isValue(Predicate<String> predicate) {
+    return expr -> TextUtils.matchesValue(expr, predicate).isTrue();
+  }
```

Comment:
If this is merged, I will use it too :D

Reply:
I did this before our daily standup, but as discussed I will move it to a new class PredicateUtils or something similar.